data/json/construction.json

```diff
@@ -2390,7 +2390,8 @@
             [ [ "2x4", 6 ], [ "stick", 6 ] ],
             [ [ "withered", 12 ], [ "straw_pile", 12 ] ]
         ],
-        "pre_terrain" : "t_pit",
-        "post_terrain" : "t_rootcellar"
+        "pre_note":  "You need a deep pit to construct a root cellar.",
```

Comment:
This is helpful but not exactly [what I was referring to.](https://github.com/CleverRaven/Cataclysm-DDA/pull/24083#discussion_r197656156)

A "root cellar's" purpose isn't evident in its name. A user would have to construct the terrain and then <kbd>e</kbd>xamine it (and read [the description](https://github.com/CleverRaven/Cataclysm-DDA/pull/24083/files#diff-f8cfb96b38db5f164557489893669370R6850)) in order to determine it's purpose.

The `pre_note` would be a good place to make that evident from the beginning.

For instance, `You first need a deep pit to construct a root cellar, and enable the storage of food in a cold environment`.

Wordy, but you get the gist :)

Reply:
@Leland 
No, it would'nt. The sole purpose of a pre-note is to give some info on extra costruction/deconstruction requrements.
What you ask is actualy true for all constructions. Solution to this would be expanding construction menu with the description of a constructed item, so you will get the idea of what actualy you are constructing.

This deserves a separate PR and i will try to approach it, as it bothers me too.